### PR TITLE
LAWLIFT connector

### PR DESCRIPTION
**Pull request by makaveli1313**
---
When submitting a connector, please check the following conditions for your PR.

- [x ] I attest that the connector works and I verified by deploying and testing all the operations.
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a logo.

### CODE REVIEW

File: certified-connectors/LAWLIFT/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,660 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "LAWLIFT",
+    "description": "LAWLIFT is a company focused on document automation. The editor, which has become known for its ease of use, has won several awards. Customers are large and medium-sized companies, numerous big law firms, legal departments and notary offices. For more information visit : www.lawlift.com .",
+    "version": "V1",
```
Comment (sriyen-msft) [inline certified-connectors/LAWLIFT/apiDefinition.swagger.json:6]: @makaveli1313 Can you please keep the version like this as we can keep sub versions as well like this:

`"version": "1.0.0"`

File: certified-connectors/LAWLIFT/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,660 @@
...
+          "500": {
+            "description": "Internal server error"
+          }
+        },
+        "security": [
+          {
+            "API Key": []
+          }
+        ],
+        "summary": "LAWLIFT notification trigger",
```
Comment (sriyen-msft) [inline certified-connectors/LAWLIFT/apiDefinition.swagger.json:344]: @makaveli1313 Usually summaries of triggers would start with "When .." or "Trigger..", Can you please update it

File: certified-connectors/LAWLIFT/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,660 @@
...
+          "type": "string",
+          "format": "date-time",
+          "title": "Template   created timestamp",
+          "description": "The timestamp that the template was created.",
+          "x-ms  -visibility": "advanced"
+        },
+        "updatedAt": {
+          "type": "string",
+          "format": "date-time",
+          "title": "Template   updated timestamp",
```
Comment (sriyen-msft) [inline certified-connectors/LAWLIFT/apiDefinition.swagger.json:481]: @makaveli1313 Please remove extra spaces

File: certified-connectors/LAWLIFT/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,660 @@
...
+          "description": "The timestamp that the template was updated.",
+          "x-ms  -visibility": "advanced"
```
Comment (sriyen-msft) [inline certified-connectors/LAWLIFT/apiDefinition.swagger.json:483]: @makaveli1313 This is a syntax error, Please remove the spaces in tags. This applies anywhere in the connector files.

File: certified-connectors/LAWLIFT/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,660 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "LAWLIFT",
```
Comment (sriyen-msft) [inline certified-connectors/LAWLIFT/apiDefinition.swagger.json:4]: @makaveli1313 It is not allowed to keep the connector's title in capital letters, Can you please use camel-case for your connector's name anywhere in the connector artifacts

File: certified-connectors/LAWLIFT/apiProperties.json [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+          "tooltip": "Provide your API Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#000000",
```
Comment (sriyen-msft) [inline certified-connectors/LAWLIFT/apiProperties.json:18]: @makaveli1313 Please make sure that the background of your icon is `#000000`

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by sriyen-msft** [comment]
@makaveli1313 Please remove icon from the PR

File: certified-connectors/LAWLIFT/apiProperties.json (hunk 6 above)
Comment (makaveli1313) [reply certified-connectors/LAWLIFT/apiProperties.json:18]: > @makaveli1313 Please remove icon from the PR

Thank you for your answers. We should move it to our folder? It's not clear from the documentation.

**Comment by makaveli1313** [comment]
@sriyen-msft If there is anything else we need to do, please let us know.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**